tests/integration/states/azurerm/web/test_function_app.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+import random
+import string
+import os
+import tempfile
+import pathlib
+import zipfile
+
+
+@pytest.fixture(scope="session")
```

Comment:
why `session` vs `module` fixtures? will other tests use these?

Reply:
Module gets called for every (test) module within the file, while session is called once for the overall file. If the scope is "module," then a new randomly generated name is used for each module and the tests break. If it was something not randomly generated I could set the scope as "module."